observability/distr_tracing/distr-tracing-rn.adoc

```diff
@@ -12,9 +12,166 @@ You can use the {TempoName} xref:../otel/otel-forwarding-telemetry-data.adoc#ote
 
 include::snippets/distr-tracing-and-otel-disclaimer-about-docs-for-supported-features-only.adoc[]
 
-// git rebase marker 1
+[id="distr-tracing_3-4_{context}"]
+== Release notes for {DTProductName} 3.4
 
-// git rebase marker 2
+This release of the {DTProductName} includes the {TempoName} and the deprecated {JaegerName}.
+
+[id="distr-tracing_3-4_cves_{context}"]
+=== CVEs
+
+This release fixes the following CVEs:
+
+* link:https://access.redhat.com/security/cve/CVE-2024-21536[CVE-2024-21536]
+* link:https://access.redhat.com/security/cve/CVE-2024-43796[CVE-2024-43796]
+* link:https://access.redhat.com/security/cve/CVE-2024-43799[CVE-2024-43799]
+* link:https://access.redhat.com/security/cve/CVE-2024-43800[CVE-2024-43800]
+* link:https://access.redhat.com/security/cve/CVE-2024-45296[CVE-2024-45296]
+* link:https://access.redhat.com/security/cve/CVE-2024-45590[CVE-2024-45590]
+* link:https://access.redhat.com/security/cve/CVE-2024-45811[CVE-2024-45811]
+* link:https://access.redhat.com/security/cve/CVE-2024-45812[CVE-2024-45812]
+* link:https://access.redhat.com/security/cve/CVE-2024-47068[CVE-2024-47068]
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=2312579[Cross-site Scripting (XSS) in `serialize-javascript`]
+
+[id="distr-tracing_3-4_tempo-release-notes_{context}"]
+=== {TempoName}
+
+The {TempoName} 3.4 is provided through the link:https://catalog.redhat.com/software/containers/rhosdt/tempo-operator-bundle/642c3e0eacf1b5bdbba7654a/history[{TempoOperator} 0.14.1].
+
+[NOTE]
+====
+The {TempoName} 3.4 is based on the open source link:https://grafana.com/oss/tempo/[Grafana Tempo] 2.6.1.
+====
+
+////
+[id="distr-tracing_3-4_tempo-release-notes_technology-preview-features_{context}"]
+==== Technology Preview features
+
+This update introduces the following Technology Preview feature:
+
+* ???.
+
+:FeatureName: Each of these features
+include::snippets/technology-preview.adoc[leveloffset=+1]
+////
+
+[id="distr-tracing_3-4_tempo-release-notes_new-features-and-enhancements_{context}"]
+==== New features and enhancements
+
+This update introduces the following enhancements:
+
+* The monitor tab in the Jaeger UI for TempoStack instances uses a new default metrics namespace: `traces.span.metrics`. Before this update, the Jaeger UI used an empty namespace. The new `traces.span.metrics` namespace default is also used by the OpenTelemetry Collector 0.112.0. Users can set the empty value for the metrics namespace by using the following field in the `TempoStack` custom resouce: `spec.template.queryFrontend.monitorTab.redMetricsNamespace: ""`.
++
+[WARNING]
+====
+This is a breaking change.
```

Comment:
@andreasgerstmayr, WDYT? Or feel free to add a suggestion. 
```suggestion
This is a breaking change. Users who are using both the {TempoName} and {OTELName} must upgrade to the {OTELName} 3.4 before upgrading to the {TempoName} 3.4.
```

Reply:
sgtm

Reply:
TY, applying.